### PR TITLE
docs: :memo: wrote a post on my experiences in the Science Cluster conference

### LINKED ISSUE #31: Post on experience of Science cluster conference

### PR DESCRIPTION
**Pull request by lwjohnst86**
## Description

Finally wrote my experiences on the data science symposium.

Closes #31

## Reviewer Focus

This PR doesn't strictly need a review, but reviewing/proofreading it would be helpful. Mostly, just so you all can read about my experiences of it.

## Checklist

- [x] Ran spell-check
- [x] Formatted Markdown
- [x] Rendered website locally

### CODE REVIEW

File: posts/symposium-science-cluster.qmd [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    was very little of that.
+
+## Interesting insights and ideas
+
+I did get some interesting insights and ideas from the symposium that I
+think might be useful to incorporate or consider in some way in the
+Seedcase Project:
+
+-   There are a lot of -omic based projects, so we need to consider how
+    those types of data will be input into Seedcase software.
```
Comment (signekb) [inline posts/symposium-science-cluster.qmd:137]: Maybe elaborate a bit on this and add a few examples of what types of data you're thinking of here? Is it other data types? Relations in the data? Something else?

File: posts/symposium-science-cluster.qmd [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+-   One presentation was on tracking energy use when running analyses on
+    the server. It made me consider how much energy our work will use
+    and what we could potentially do to lower it.
```
Comment (signekb) [inline posts/symposium-science-cluster.qmd:141]: Good point. How would we go about that? 
Maybe stuff like:
- Ensuring that the same data isn't added multiple times (deduplication).
- Thinking about if we could compress the data to reduce data size before storing or transmitting (this also includes us thinking about how we currently plan to store the data as both`raw`, `parquet`, *and potentially* SQL).
- Making our UI minimalistic/lightweight by avoiding unnecessary visual elements that require extra processing power.

File: posts/symposium-science-cluster.qmd [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+-   Further emphasized the strong fundamental need for more data
```
Comment (signekb) [inline posts/symposium-science-cluster.qmd:143]:
```suggestion
-   Further emphasize the strong fundamental need for more data
```

File: posts/symposium-science-cluster.qmd [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+    engineering practices, even though the demand for these skills,
+    knowledge, and tools hasn't yet reached a conscious level in most
+    researchers minds.
+
+-   Even projects that are doing similar work as ours towards building
+    data infrastructures, they are closed source or not publicly
```
Comment (signekb) [inline posts/symposium-science-cluster.qmd:149]:
```suggestion
    data infrastructures are closed source or not publicly
```

File: posts/symposium-science-cluster.qmd [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+    accessible and it isn't obvious how it can be used in other
+    contexts. Which highlights the niche that Seedcase is aiming to fill
```
Comment (signekb) [inline posts/symposium-science-cluster.qmd:151]:
```suggestion
    contexts. This highlights the niche that Seedcase is aiming to fill
```

File: posts/symposium-science-cluster.qmd [hunk 6]
```diff
@@ -0,0 +1,189 @@
...
+    in the research community.
```
Comment (signekb) [inline posts/symposium-science-cluster.qmd:152]: But also beyond research, right? E.g., small-to-medium sized companies

**Review by signekb — CHANGES_REQUESTED** [review]
This made me smile bc I **might** have heard some of these points before 😁

**Review by signekb — CHANGES_REQUESTED** [review]
This made me smile bc I **might** have heard some of these points before 😁

File: posts/symposium-science-cluster.qmd (hunk 2 above)
Comment (martonvago) [reply posts/symposium-science-cluster.qmd:141]: I don't have anything useful to add, but I also find this question very interesting.

File: posts/symposium-science-cluster.qmd [hunk 7]
```diff
@@ -0,0 +1,189 @@
...
+-   The presentations on communication, specifically on accessibility
+    and on using effective graphics were particularly interesting. This
+    is especially important since they bring awareness to these issues
+    that are often overlooked or ignored in scientific conferences and
+    similar types of events. Coming from someone (me) who has difficulty
+    differentiating colors, I really appreciated the emphasis on
+    accessibility.
+
+-   One of the final presentations by Lars Juhl Jensen was especially
+    good, since he showed how to give an engage and informative
```
Comment (martonvago) [inline posts/symposium-science-cluster.qmd:173]:
```suggestion
    good, since he showed how to give an engaging and informative
```

**Review by martonvago — CHANGES_REQUESTED** [review]
Super interesting 😁 ! (Just some typos)

File: posts/symposium-science-cluster.qmd (hunk 1 above)
Comment (lwjohnst86) [reply posts/symposium-science-cluster.qmd:137]: Hmm, this is something other researchers would know about. -omics is the type of data I am meaning. Genomics, proteomics, metabolomics, etc. I can add a clarification there.

Comment (lwjohnst86) [inline posts/symposium-science-cluster.qmd:137]:
```suggestion
    those types of -omic data like genomics or metabolomics will be input into Seedcase software.
```

File: posts/symposium-science-cluster.qmd (hunk 2 above)
Comment (lwjohnst86) [reply posts/symposium-science-cluster.qmd:141]: Yea, I don't really know how to go about it yet either. But it is something to keep in mind as we go along ☺️

File: posts/symposium-science-cluster.qmd (hunk 6 above)
Comment (lwjohnst86) [inline posts/symposium-science-cluster.qmd:152]:
```suggestion
    in the research community (and also in small- to medium-sized companies).
```

File: posts/symposium-science-cluster.qmd [hunk 8]
```diff
@@ -0,0 +1,189 @@
...
+## Interesting insights and ideas
+
+I did get some interesting insights and ideas from the symposium that I
+think might be useful to incorporate or consider in some way in the
+Seedcase Project:
+
+-   There are a lot of -omic based projects, so we need to consider how
+    those types of data will be input into Seedcase software.
+
+-   One presentation was on tracking energy use when running analyses on
```
Comment (lwjohnst86) [inline posts/symposium-science-cluster.qmd:139]:
```suggestion
-   One presentation by Alexander Henriksen was on tracking energy use when running analyses on
```

File: posts/symposium-science-cluster.qmd [hunk 9]
```diff
@@ -0,0 +1,189 @@
...
+    (or even explicit) principles, values, and designs. Building up
+    these principles and designs takes time, so it can seem that we are
+    slow with producing software. But with these strong cultural and
+    engineering foundations, we establish a base to do effective,
+    rigorous, and high-quality work that will last well into the future.
+
+There were a few presentations I really appreciated and enjoyed, which
+also happened to be about how to give effective presentations!
+
+-   The presentations on communication, specifically on accessibility
```
[the remark this answers is not in the document]
Comment (lwjohnst86) [reply posts/symposium-science-cluster.qmd:164]:
```suggestion
-   The presentations by Eleonora Nigro and Anika Gupta on communication, specifically on accessibility via visuals
```

File: posts/symposium-science-cluster.qmd [hunk 10]
```diff
@@ -0,0 +1,189 @@
...
+
+-   One of the final presentations by Lars Juhl Jensen was especially
+    good, since he showed how to give an engage and informative
+    presentation. His style emphasized using simple slides, mostly just
+    a few words or a single image, while keeping the entire focus on him
+    and his speaking and presence. While I've heard some say (or fear?)
+    that there isn't enough "content/results", the major advantage to
+    this style is that it is more accessible to a wider audience of
+    people from different fields of research or work, countries and
+    proficiency in English, and in neurodiverse capabilities.
```
[the remark this answers is not in the document]
Comment (lwjohnst86) [reply posts/symposium-science-cluster.qmd:180]: Unfortunately, live presentations are currently a terrible format for people with hearing difficulties 😢 if they are recorded and a closed caption subtitles are added, then it works, but while there, it isn't great.

File: posts/symposium-science-cluster.qmd (hunk 1 above)
Comment (signekb) [reply posts/symposium-science-cluster.qmd]: Well, maybe not *all* researchers 😄

Comment (lwjohnst86) [reply posts/symposium-science-cluster.qmd]: *health* researchers to be precise 😝

**Pull request merged**